test/interaction/Issue6082.agda

```diff
@@ -2,7 +2,7 @@
 -- Reflected syntax does not know about projections,
 -- so when translating to abstract syntax, need to recover them.
 
-{-# OPTIONS --postfix-projections -vtc.reconstruct:50 #-}
+{-# OPTIONS --postfix-projections #-}
```

Comment:
Does the `Issue6082.out` file needs update?

Reply:
No, I removed reconstruction from elaborate-and-give in the same commit that I (accidentally) included this `-v` on. So it never had an effect